server/src/services/person.service.ts

```diff
@@ -717,4 +719,25 @@ export class PersonService extends BaseService {
       height: newHalfSize * 2,
     };
   }
+
+  async tagFace(auth: AuthDto, dto: TagFaceDto): Promise<void> {
+    await this.requireAccess({ auth, permission: Permission.PERSON_READ, ids: [dto.personId] });
```

Comment:
Maybe you could argue this should require `PERSON_WRITE`?

Reply:
PERSON_WRITE or PERSON_CREATE is to create a new person in the person table. Here you need to be able to read the person list in order to create a new entry in the asset_faces table

Reply:
probably should be a new permission like `Permission.FACE_CREATE`